unified_planning/engines/mixins/simulator.py

```diff
@@ -152,3 +171,31 @@ def _get_events(
     @staticmethod
     def is_simulator():
         return True
+
+    def is_goal(self, state: "up.model.ROState") -> bool:
+        """
+        Returns True if the given state satisfies the goals of the problem.
+        :param state: the State in which the problem goals are evaluated.
+        :return: True if the evaluation of every goal is True, False otherwise.
+        """
+        return self._is_goal(state)
+
+    def _is_goal(self, state: "up.model.ROState") -> bool:
+        raise NotImplementedError
```

Comment:
Why not giving a default implementation based on `get_unsatisfied_goals`? one can always override this ;-)

Reply:
Same for `is_applicable`, maybe